src/components/post-card/post-card.module.scss

```
@@ -1,112 +1,229 @@
+	--card_snippet_color: var(--foreground_emphasis-medium);
+	--card_detail-row_details_color: var(--foreground_emphasis-high);
+	--card_detail-row_icon_color: var(--foreground_disabled);
+	--card_detail-row_dot_color: var(--foreground_disabled);
+
+	--card_background-color: var(--transparent);
+	--card_background-color_hovered: var(--surface_primary_emphasis-low);
+	--card_background-color_pressed: var(--surface_primary_emphasis-low);
+	--card_background-color_focused: var(--background_focus);
+
+	@include from($desktopSmall) {
+		--card_padding: var(--spc-6x);
+		--card_corner-radius: var(--corner-radius_xl);
+
+		--card_detail-container_gap: var(--spc-2x);
+		--card_detail-container_padding-vertical: var(--spc-3x);
+
+		/* Prominent cards have bigger spacing */
+		--card_detail-container_padding-vertical_prominent: var(--spc-4x);
+	}
+}
 
-	padding: var(--spc-6x);
+.postBase {
+	border-radius: var(--card_corner-radius);
+	border: var(--card_border-width) solid var(--card_border-color);
+	background: var(--card_background-color);
+	overflow: hidden;
+}
 
-	cursor: pointer;
+.postBase:hover {
+	border-color: var(--card_border-color_hovered);
+	background: var(--card_background-color_hovered);
+}
 
-	&:hover {
-		background-color: var(--surface_primary_emphasis-low);
-		border-color: transparent;
-	}
+.postBase:active {
+	border-color: var(--card_border-color_pressed);
+	background: var(--card_background-color_pressed);
+}
 
-	&:active {
-		background-color: var(--surface_primary_emphasis-low);
-		border-color: var(--surface_primary_emphasis-high);
+@supports selector(:has(*)) {
+	.postBase:has(:focus-visible) {
+		border-color: var(--card_border-color_focused);
+		background: var(--card_background-color_focused);
 	}
+}
 
-	&:focus {
-		background-color: var(--background_focus);
-		border-color: var(--focus-outline_primary);
+@supports not selector(:has(*)) {
+	.postBase:focus-within {
+		border-color: var(--card_border-color_focused);
+		background: var(--card_background-color_focused);
 	}
 }
 
-.cardExpanded {
-	border: var(--border-width_xl) solid var(--surface_primary_emphasis-low);
-	border-radius: var(--corner-radius_xl);
+.postContainer {
+	padding: var(--card_padding);
+}
 
-	overflow-wrap: break-word;
-	max-width: 100%;
+.postHeaderBase {
+	color: var(--card_title_color);
+	margin: 0;
 	text-decoration: none;
-	@include transition(background-color border-color);
+}
+
+.postHeaderBase > h2 {
+	margin: 0;
+}
 
-	padding: var(--spc-6x);
+.postDataContainer {
+	padding: var(--card_detail-container_padding-vertical) 0;
 	display: flex;
 	flex-direction: column;
-	position: relative;
-	justify-content: flex-end;
-	overflow: hidden;
-
-	cursor: pointer;
+	gap: var(--card_detail-container_gap);
+}
 
-	&:hover {
-		background-color: var(--surface_primary_emphasis-low);
-		border-color: transparent;
-	}
+.authorList {
+	list-style: none;
+	margin: 0;
+	padding: 0;
+	display: flex;
+	flex-direction: row;
```

```suggestion
	flex-direction: row;
	flex-wrap: wrap;
	gap: var(--card_detail-row_gap);
```

I believe this is needed to properly handle multiple authors in the list (currently they don't wrap, and have no spacing between authors).